app/mailers/referee_mailer.rb

```diff
@@ -24,8 +25,9 @@ def reference_request_chaser_email(application_form, reference)
     @reference = reference
     @candidate_name = application_form.full_name
 
+    @token = reference.update_token!
```

Comment:
It's not obvious to me why we are turning this to an instance variable, what's up? Not blocking anyway.

Reply:
It's used inside the email. I'll look into tidying up the emails in a separate PR.